.travis.yml

```diff
@@ -15,4 +15,7 @@ script:
 branches:
   only:
     - master
+    
```

Comment:
Please update coverage threshold numbers in jacoco.sbt

Reply:
Done.